protocols/1291-griffith-university-part1/cell_culture_part1.ot2.py

```diff
@@ -54,22 +60,41 @@ def run_custom_protocol(
                   for index in range(2)]
     m300.distribute(medium_volume, medium, multi_dest, disposal_vol=0)
 
-    p300.start_at_tip(tiprack.wells('A3'))
+    p300.start_at_tip(tiprack.wells('A4'))
```

Comment:
Is this the second time they have changed the tiprack location? Maybe we should just make it a custom variable.